Core/MIPS/ARM64/Arm64IRCompLoadStore.cpp

```diff
@@ -38,20 +38,173 @@ namespace MIPSComp {
 using namespace Arm64Gen;
 using namespace Arm64IRJitConstants;
 
+static int IROpToByteWidth(IROp op) {
+	switch (op) {
+	case IROp::Load8:
+	case IROp::Load8Ext:
+	case IROp::Store8:
+		return 1;
+
+	case IROp::Load16:
+	case IROp::Load16Ext:
+	case IROp::Store16:
+		return 2;
+
+	case IROp::Load32:
+	case IROp::Load32Linked:
+	case IROp::Load32Left:
+	case IROp::Load32Right:
+	case IROp::LoadFloat:
+	case IROp::Store32:
+	case IROp::Store32Conditional:
+	case IROp::Store32Left:
+	case IROp::Store32Right:
+	case IROp::StoreFloat:
+		return 4;
+
+	case IROp::LoadVec4:
+	case IROp::StoreVec4:
+		return 8;
```

Comment:
A Vec4 is 16 bytes - though if this represents the actual size of the writes (paired stores?), rather than the granularity of the data, it needs a comment.

Reply:
Oops, no, that's just a typo.  Was for the forced alignment of LDR, which does need to be 16.

-[Unknown]